index.html

```diff
@@ -6,8 +6,22 @@
     <title>Resque_a_pet</title>
     <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="assets/css/fonts.css" type="text/css"/>
+    <link rel="stylesheet" href="assets/css/main.css" type="text/css"/>
+    <link rel="stylesheet" href="assets/css/heading.css" type="text/css"/>
+    <link rel="stylesheet" href="assets/css/reset.css" type="text/css"/>
 </head>
 <body>
-    
+ <div class="container">
+  <header>
+    <div class="header-logo"style="background-image: url('assets/images/logo.png');"></div>
+    <div class="header-photo"style="background-image: url('assets/images/the-dog-white-and-brown.jpeg');"></div>
+      <div class="header-social-media-wrapper">
+       <i class="header-social-media-fb"></i>
```

Comment:
Źle. Miały być dwie klasy.
`<i class="header-social-media-icon fb"></i>`

Wtedy możesz zrobić tak:
```
.header-social-media-icon{
font-size: xxx;
color: #fff;
transition: xxx;
}
```

```
.header-social-media-icon.fb:hover{
color:blue;
}
```

Dzięki temu wszystkie 3 ikony mają jedną i tą samą klasę (raz ustawiasz im style), a dodatkowa, druga klasa (fb/tw/ig) może dodać unikalne wartości lub nadpisać którąś


W przypadku Twojego kodu trzeba zrobić tak:
```
.header-social-media-fb{
font-size: xxx;
color: #fff;
transition: xxx;
}
.header-social-media-tw{
font-size: xxx;
color: #fff;
transition: xxx;
}
.header-social-media-ig{
font-size: xxx;
color: #fff;
transition: xxx;
}
```

lub po prostu:
```
.header-social-media-wrapper i{
costam: costam;
}
```
- ale tak prosiłem zeby nie robić. Mamy klasy po to, żeby ich używać :)

Reply:
Zrobione